java/pom.xml

```diff
@@ -242,6 +242,12 @@
       <version>20211205</version>
     </dependency>
 
+    <dependency>
```

Comment:
We shouldn't compile against a JDBC connector for a particular database. It would be better to require the users to add the connector JARs to their job at submit time and parametrize the driver class name. That's what the [existing `JDBCTo*`](https://github.com/GoogleCloudPlatform/dataproc-templates/tree/main/java/src/main/java/com/google/cloud/dataproc/templates/jdbc) connectors do.

Reply:
Removed the dependency of jar file from pom.xml. Now onwards passing the jar file location via command line argument using export would be required.